src/Nethermind/Nethermind.Synchronization.Test/BlockDownloaderTests.cs

```diff
@@ -619,6 +619,13 @@ public bool Validate(BlockHeader header, bool isUncle, [NotNullWhen(false)] out
             error = null;
             return true;
         }
+
+        public bool ValidateBodyAgainstHeader(BlockHeader header, BlockBody toBeValidated, [NotNullWhen(false)] out string? errorMessage)
+        {
+            Thread.Sleep(1000);
```

Comment:
I see it follows the construction of others, but still, `Thread.Sleep` makes me wonder what is going on here.

Reply:
Agree, but it's as you mentioned: follow the construction of others.